test/specs/features.js

```diff
@@ -32,4 +32,31 @@ describe("Feature tests", () => {
     const h2header = $("h2=Restaurants near W13 3RH");
     expect(h2header).toBeDisplayed();
   });
+
+  it("Has the ability for user to click dropdown menu", () => {
+    browser.url(homepageUrl);
+    const input = $('[name="postcode"]');
+    input.addValue("W13 3RH");
+    const submitButton = $(".submit-button");
+    submitButton.click();
+    const selectBox = $("#filter");
+    expect(selectBox.isClickable());
+  });
+
+  it("Has the ability to change the restaurant order based on rating or nearest to postcode", () => {
+    browser.url(homepageUrl);
+    const input = $('[name="postcode"]');
+    input.addValue("W13 3RH");
+    const submitButton = $(".submit-button");
+    submitButton.click();
+    const selectBox = $("#filter");
+    const elem = $("h2=A 2 Zee's");
+    expect(elem).toBeVisibleInViewport();
+    selectBox.selectByAttribute("value", "rating");
+    const elemNotVisible = $("h2=A 2 Zee's");
+    expect(elemNotVisible).not.toBeVisibleInViewport();
+    selectBox.selectByAttribute("value", "nearest");
+    const elemVisible = $("h2=A 2 Zee's");
+    expect(elemVisible).toBeVisibleInViewport();
```

Comment:
What I want to know about for the search results page:
- Know that I'm on the right route
- Main page information is correct (headings, etc)
- All the mocked data is as expected (title, rating, location, etc)
- That the mocked data is ordered as expected
- That I can filter/change the order of the mocked data as expected